packages/runtime/container-runtime/src/test/gc/gcUnitTestHelpers.ts

```diff
@@ -3,12 +3,28 @@
  * Licensed under the MIT License.
  */
 
-import { ConfigTypes, IConfigProviderBase } from "@fluidframework/core-interfaces";
+import { ConfigTypes } from "@fluidframework/core-interfaces";
 import { ReadAndParseBlob } from "@fluidframework/runtime-utils";
 
-export const configProvider = (settings: Record<string, ConfigTypes>): IConfigProviderBase => ({
-	getRawConfig: (name: string): ConfigTypes => settings[name],
-});
+/**
+ * Creates a test config provider with the ability to set configs values and clear all config values.
+ * @internal
+ */
+export const createTestConfigProvider = () => {
+	const settings: Record<string, ConfigTypes> = {};
+	return {
+		getRawConfig: (name: string): ConfigTypes => settings[name],
+		set: (key: string, value: ConfigTypes) => {
+			settings[key] = value;
+		},
+		clear: () => {
```

Comment:
I think with this setup you actually could do `settings = {}` but I'll have to check in TS Playground to be sure.